keanu-python/keanu/vertex.py

```diff
@@ -1,18 +1,67 @@
 import numpy as np
 import numbers
 
+import keanu as kn
 from keanu.context import KeanuContext
 from keanu.base import JavaObjectWrapper
-from keanu.const import Const
```

Comment:
I had to remove this to avoid a circular import Vertex <--> Const

Reply:
You could just also import it inline

Reply:
Good idea. In this case I will leave it since doing `import keanu as kn` instead breaks the circular dependency, and I use it elsewhere in this file too.